packages/schematics/src/command-line/shared.ts

```diff
@@ -211,7 +211,12 @@ function recursiveMtime(dirName: string) {
 }
 
 function mtime(f: string): number {
-  return fs.fstatSync(fs.openSync(f, 'r')).mtime.getTime();
+  let fd = fs.openSync(f, 'r');
```

Comment:
did some tests, had some file lock issues too in vscode that are gone now (e.g. couldn't revert) 👍

... to be lint friendly, this should be:
```javascript
const fd = fs.openSync(f, 'r');
```